topi/include/topi/cuda/injective.h

```diff
@@ -33,21 +33,26 @@ namespace topi {
 using namespace tvm;
 
 namespace cuda {
+
 /*!
-* \brief Schedule a given injective operation.
-*
-* \param target The target to generate a schedule for.
-* \param op The operation representing the injective operation.
-* \param s The schedule to apply this scheduling to
-*/
-inline void ScheduleInjectiveOp(const Target &target, Operation op, Schedule s) {
-  auto x = op.output(0);
-  auto fused = detail::Fuse(s[x], s[x]->op.as<ComputeOpNode>()->axis);
+ * \brief Updates an existing schedule for the given injective ops.
+ *
+ * \param target The target to update the schedule for.
+ * \param sch The schedule to update.
+ * \param out The tensor representing the injective op.
+ * 
+ * \return The updated schedule.
+ */
+inline Schedule schedule_injective_from_existing(const Target& target,
+                                                 Schedule sch,
+                                                 const Tensor& out) {
+  auto fused = detail::Fuse(sch[out], sch[out]->op.as<ComputeOpNode>()->axis);
   auto num_thread = target->max_num_threads;
```

Comment:
Can be replaced by  
```
auto target = Target::Current(false);
auto num_thread = target->max_num_threads;
```

Reply:
I originally tried to do this, but there was a GPU unit test that was failing :( It seemed that there wasn't a global target set, rather it was passed directly in `schedule_injective`.

This is probably also why all other generic `schedule_*` functions take the target as an argument.

Reply:
I see. I think this can be fixed by adding "with tvm.target.create(device)", but may be for another PR.